libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/OfflineError.kt

```diff
@@ -0,0 +1,5 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+data class OfflineError(
+    val message: String
```

Comment:
Should we add anything special here as this is a `public` class and can be used from Java?

Reply:
nope, `data class` is a class but with overrides `equal` and `hashCode` func

Reply:
Yeah I know 😅 but now `String` is not-null, wondering if we should add `@NotNull` annotation for Java clients.

Reply:
it's actually NonNull now for Java's users :)

Reply:
Ohh yeah, that's right the compiler does actually that 👍